operator/pkg/tasks/init/upload.go

```diff
@@ -85,10 +96,24 @@ func runUploadAdminKubeconfig(r workflow.RunData) error {
 	}
 	data.SetControlplaneConifg(config)
 
-	klog.V(2).InfoS("[upload-config] Successfully created secret of karmada apiserver kubeconfig", "karmada", klog.KObj(data))
+	klog.V(2).InfoS("[UploadAdminKubeconfig] Successfully created secret of karmada apiserver kubeconfig", "karmada", klog.KObj(data))
 	return nil
 }
 
+func getNodePortFromAPIServerService(service *corev1.Service) int32 {
+	var nodePort int32
+	if service.Spec.Type == corev1.ServiceTypeNodePort {
+		for _, port := range service.Spec.Ports {
+			if port.Name != "client" {
```

Comment:
May I ask a question why we need this logic?

Reply:
@Poor12  Becase we need to get the karmada-apiserver nodeport that be allocated by kubernetes. and port name of its service is `client`, so in here we need find corresponding port.NodePort.

https://github.com/karmada-io/karmada/blob/master/operator/pkg/controlplane/apiserver/mainfests.go#L125

Reply:
thanks for your review.

Reply:
Got it. BTW, I just found that we use `kube-apiserver` as the label value of `karmada-apiserver`, could you please reminder me why we do not use `karmada-apiserver`.
https://github.com/karmada-io/karmada/blob/7b1ddc9ee18d8bd8704ebc90db98f14ab575c92b/operator/pkg/controlplane/apiserver/mainfests.go#L130

Reply:
I'm with you on this point. it should be karmada-apiserver

Reply:
@Poor12  Because the  karmada image repository is "registry.k8s.io/kube-apiserver",  we use the same constant `KarmadaAPIServer` to image and lebal key.
https://github.com/karmada-io/karmada/blob/master/operator/pkg/tasks/init/wait.go#L18

Reply:
I feel like it's best not to reuse this value to the image and label value. They don't mean the same thing.

Reply:
I'm with you on this.